### PR TITLE
runtime: update default machine type to q35

### PR DESCRIPTION
**Pull request by liubin**
The machine type of pc is deleted, generated configuration
should update too.

Fixes: #2085

Signed-off-by: bin <bin@hyper.sh>

### CODE REVIEW

**Review by fidencio — APPROVED** [review]
lgtm, thanks @liubin!

**Review by Jakob-Naucke — APPROVED** [review]

**Comment by liubin** [comment]
> /retest-vfio

@fidencio , should we delete these two lines too?

https://github.com/kata-containers/tests/blob/c99b090a7e765c4b5ba1ac881226a5d1bf6324fe/Makefile#L168-L169

```
	bash -f functional/vfio/run.sh -s false -p qemu -m pc -i image
	bash -f functional/vfio/run.sh -s true -p qemu -m pc -i image
```

**Comment by fidencio** [comment]
> > /retest-vfio
> 
> @fidencio , should we delete these two lines too?
> 
> https://github.com/kata-containers/tests/blob/c99b090a7e765c4b5ba1ac881226a5d1bf6324fe/Makefile#L168-L169
> 
> ```
> 	bash -f functional/vfio/run.sh -s false -p qemu -m pc -i image
> 	bash -f functional/vfio/run.sh -s true -p qemu -m pc -i image
> ```

Yes, I think we should!

@liubin, mind to add that to your PR?

**Comment by liubin** [comment]
@fidencio test cases is in tests repo, I will do this in that repo.

**Comment by fidencio** [comment]
Hmm. The metrics do not seem to be very stable after the change.  Let me rerun that CI and, @marcel-apf, we may need to update the metrics again after a few runs getting to feel how much it decreased.

**Comment by marcel-apf** [comment]
@fidencio sure, I'll take care of that

File: src/runtime/arch/amd64-options.mk [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 # Intel x86-64 settings
 
-MACHINETYPE := pc
+MACHINETYPE := q35
```
Comment (marcel-apf) [inline src/runtime/arch/amd64-options.mk:8]: I wonder how I missed that :( , anyway thanks for the fix!

**Comment by marcel-apf** [comment]
> > > /retest-vfio
> > 
> > 
> > @fidencio , should we delete these two lines too?
> > https://github.com/kata-containers/tests/blob/c99b090a7e765c4b5ba1ac881226a5d1bf6324fe/Makefile#L168-L169
> > ```
> > 	bash -f functional/vfio/run.sh -s false -p qemu -m pc -i image
> > 	bash -f functional/vfio/run.sh -s true -p qemu -m pc -i image
> > ```
> 
> Yes, I think we should!
> 
> @liubin, mind to add that to your PR?

+1

I need to improve my "grep" skills :)

**Comment by fidencio** [comment]
> /retest-vfio
> /retest-ubuntu-metrics

Those should pass now as the tests PR for the PC removal has been merged. Let's wait and check it soon enough.

**Comment by marcel-apf** [comment]
> > /retest-vfio
> > /retest-ubuntu-metrics
> 
> Those should pass now as the tests PR for the PC removal has been merged. Let's wait and check it soon enough.

Now it makes more sense. The tests PR wasn't merged. Thanks @fidencio

**Pull request merged**